packages/client/hmi-client/src/page/Home.vue

```diff
@@ -102,81 +127,139 @@ const scroll = (direction: 'right' | 'left', event: PointerEvent) => {
 				/>
 			</div>
 		</div>
-
-		<h4>Projects</h4>
-		<div class="carousel">
-			<header>
-				<component :is="IconTime32" />
-				<h5>Recent</h5>
-			</header>
-			<IconChevronLeft32 class="chevron chevron-left" @click="scroll('left', $event)" />
-			<IconChevronRight32 class="chevron chevron-right" @click="scroll('right', $event)" />
-			<ul>
-				<!-- .slice() to copy the array, .reverse() to put new projects at the left of the list instead of the right -->
-				<li v-for="(project, index) in projects.slice().reverse()" class="card" :key="index">
-					<router-link
-						style="text-decoration: none; color: inherit"
-						:to="'/projects/' + project.id"
-						:projectId="project.id"
-					>
-						<ProjectCard :project="project"></ProjectCard>
-					</router-link>
-				</li>
-			</ul>
-		</div>
-		<!-- Hot Topics carousel -->
-		<div class="carousel" v-if="relevantDocuments.length > 0">
+		<!-- New project modal -->
+		<Teleport to="body">
+			<Modal
+				v-if="isNewProjectModelVisible"
+				class="modal"
+				@modal-mask-clicked="isNewProjectModelVisible = false"
+			>
+				<template #default>
+					<form>
+						<label for="new-project-name">Project Name</label>
+						<InputText id="new-project-name" type="text" v-model="newProjectName" />
+
+						<label for="new-project-description">Project Purpose</label>
+						<Textarea id="new-project-description" rows="5" v-model="newProjectDescription" />
+					</form>
+				</template>
+				<template #footer>
+					<footer>
+						<Button @click="createNewProject">Create Project</Button>
+						<Button class="p-button-secondary" @click="isNewProjectModelVisible = false"
+							>Cancel</Button
+						>
+					</footer>
+				</template>
+			</Modal>
+		</Teleport>
+		<section class="projects">
 			<header>
-				<h5>Latest on {{ relevantSearchTerm }}</h5>
+				<h3>Projects</h3>
+				<Button
+					icon="pi pi-plus"
+					label="New project"
+					@click="isNewProjectModelVisible = true"
+				></Button>
 			</header>
-			<IconChevronLeft32 class="chevron chevron-left" @click="scroll('left', $event)" />
-			<IconChevronRight32 class="chevron chevron-right" @click="scroll('right', $event)" />
-			<ul>
-				<li v-for="(document, index) in relevantDocuments" :key="index" class="card">
-					<DocumentCard :document="document" @click="selectDocument(document)" />
-				</li>
-			</ul>
-		</div>
-		<!-- Show related documents for the top 5 projects -->
-		<div v-for="(project, index) in projectsToDisplay" :key="index" class="carousel">
+			<TabView>
+				<TabPanel header="Recents">
+					<div class="carousel">
+						<i class="pi pi-chevron-left" @click="scroll('left', $event)" />
+						<i class="pi pi-chevron-right" @click="scroll('right', $event)" />
+						<ul>
+							<li v-for="(project, index) in projects.slice().reverse()" class="card" :key="index">
```

Comment:
I realize this is copy-and-paste, but since we are doing reno in the area:
-This is not reliable, if you need a specific ordering it needs to be explicit. Neither the HMI-server nor TDS is guaranteeing ordering by created time. So we need to sort it by hand before variable assignment to "project", or the ordering needs to be requested as such.
- I suspect copying a reactive in this manner (via slice) may have quirky things down the road if we alter project properties/fields.

Reply:
I agree, this should be a computed, I don't know if we want to order them by timestamp yet. Or check with @fivegrant if the timestamp for last-updated works.

Reply:
Good enough for now.
Ping @fivegrant about adding a new last_updated field, etc.

Also, now that the projects are available on App.vue, I don't think we should fetch them again on the home page.

Reply:
TDS is ordering by id currently - typically you wouldn't want to rely on it or logical orderings.
https://github.com/DARPA-ASKEM/data-service/blob/main/tds/db/helpers.py#L34-L45


The easiest thing to do for now I think, if we want to sort it, is to hook on a sort at the end of the request

```
  projects.value = ((await API.get('/home')).data as Project[])
    .sort( ...sort logic ...);
```

Reply:
@YohannParis Since you don't know if we want to order by timestamp yet (and timestamp isn't working) should we shelve this discussion?

Reply:
Or I can add updating timestamp in the backend as part of this PR

Reply:
Actually that's in data-service which is the domain of jataware right?

Reply:
Let's create a new issue about this later.
But cleaning, ordering and those things should be done on a computed value, not on the <ul> itself.
Also the App.vue can provide the Projects now.

Reply:
@YohannParis Currently the `/home` endpoint also grabs and populates the projects' related documents. So I will have to move that functionality into `ProjectResource`

Reply:
Unless we have a separate endpoint that gets related documents only on home page... thoughts?

Reply:
As discussed, let's resolve the above in a separate task.